apis/v1alpha1/instrumentation_types.go

```diff
@@ -133,6 +133,18 @@ type Java struct {
 	// Resources describes the compute resource requirements.
 	// +optional
 	Resources corev1.ResourceRequirements `json:"resources,omitempty"`
+
+	// Extensions defines java specific extensions.
```

Comment:
@crossoverJie all extensions are copied to a single directory, therefore we should document it because one extension can override another one.

Reply:
I have added related documents.